### DISCUSSION TITLE
Suggestion for protection border for skyblock type generation

### DISCUSSION THREAD

**Discussion by Thunder-Nova** [GriefPrevention/GriefPrevention#558] [Ideas]
**As** a player,
**I want** a feature reguarding the border (the glowstone corners and such), to have the option to be at player head level, with a default option in configs to have it on or off which can still be command activated for players to have it.
**So that** it can be an option for skyblock maps (got half slabs and still the border appears under my island).

![image](https://user-images.githubusercontent.com/31967337/60763242-45254e00-a03e-11e9-9683-1aacc75de0ca.png)

as seen as those very tiny yellow dots there.

**Comment by RoboMWM** [comment]
Ayyy first feature suggestion using the story template c:

I agree, it's an even larger issue when claiming in tunnels and other indoor spaces like the nether. I've been thinking of doing something that adds at eye level.

**Comment by Thunder-Nova** [comment]
lol, I did use it, but lowered the CAP, since it feels like yelling, kept the bold.

Well, as we are into it for tunnels and such, I wonder if it is possible to make a "ghost blocks" at eye level with the glowing attribute. Or similar to seeing a ghost block thru walls, this can be an idea. Idk if it is possible since it is a block... so how about a ghost entity such as an immobile slime (it is possible to remove the AI, and make them unkillable), but same as the eye level above it has a config and a command.

**Comment by RoboMWM** [comment]
Addons may be a better bet for augmenting visualizations given that the event exists, providing freedom to experiment with particles, structure blocks, or whatever else meets the implementable imagination. I plan to make an addon of my own to address issues with visualizing a claim, albeit it's low priority on my personal list.
